dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -1051,20 +1090,44 @@ export const Explorer = ({
     setIsLiveTailOn(true);
     setToast('Live tail On', 'success');
     setIsLiveTailPopoverOpen(false);
+    setLiveTimestamp(dateMath.parse(endTime)?.utc().format(DATE_PICKER_FORMAT));
+    setLiveHits(0);
     await sleep(2000);
     const curLiveTailname = liveTailNameRef.current;
     while (isLiveTailOnRef.current === true && curLiveTailname === liveTailNameRef.current) {
       handleLiveTailSearch(startTime, endTime);
-      if (liveTailTabIdRef.current !== curSelectedTabId.current) {
-        setIsLiveTailOn(!isLiveTailOnRef.current);
+      if ((liveTailTabIdRef.current !== curSelectedTabId.current)) {
```

Comment:
I think these parentheses are unnecessary

Reply:
yeah removed one condition from there and forgot to remove the extra